packages/botonic-react/src/components/message.jsx

```diff
@@ -115,7 +115,11 @@ export const Message = props => {
   let textChildren = React.Children.toArray(children).filter(
     e => ![Button, Reply].includes(e.type)
   )
-  if (from === 'user') textChildren = textChildren.map(e => renderLinks(e))
+  if (from === 'user')
+    textChildren = textChildren.map(e => {
+      if (typeof e === 'string') return renderLinks(e)
+      return e
```

Comment:
I'll be a bit picky and suggest a simpler syntax if you please:

`return (typeof e === 'string') ? renderLinks(e) : e`

Reply:
you are a truly "picky" blinder, but I agree that it's simpler, fixed! 💯